soh/soh/SohMenuBar.cpp

```diff
@@ -82,6 +82,7 @@ std::string GetWindowButtonText(const char* text, bool menuOpen) {
     static const char* zFightingOptions[3] = { "Disabled", "Consistent Vanish", "No Vanish" };
     static const char* autosaveLabels[6] = { "Off", "New Location + Major Item", "New Location + Any Item", "New Location", "Major Item", "Any Item" };
     static const char* FastFileSelect[5] = { "File N.1", "File N.2", "File N.3", "Zelda Map Select (require OoT Debug Mode)", "File select" };
+    static const char* DekuStickCheat[3] = { "Standard", "Unbreakable", "Unbreakable + Always on Fire" };
```

Comment:
I think that `Normal` would be better than `Standard` for the menu.

Reply:
switched everything from standard to normal in https://github.com/HarbourMasters/Shipwright/pull/3192/commits/312e92773a5527c876c83b501de06dfcd763f8c5